packages/react/src/components/Forms/Select/multi-simple.tsx

```diff
@@ -5,16 +5,34 @@ import {
   SelectMultiAux,
   SubProps,
 } from ":/components/Forms/Select/multi-common";
-import { optionToString } from ":/components/Forms/Select/mono-common";
-import { SelectHandle } from ":/components/Forms/Select/index";
+import {
+  optionToString,
+  optionToValue,
+} from ":/components/Forms/Select/mono-common";
+import { Option, SelectHandle } from ":/components/Forms/Select/index";
 
 export const SelectMultiSimple = forwardRef<SelectHandle, SubProps>(
   (props, ref) => {
-    const options = React.useMemo(
-      () =>
-        props.options.filter(getMultiOptionsFilter(props.selectedItems, "")),
-      [props.selectedItems],
-    );
+    const isSelected = (option: Option) => {
+      return (
+        props.selectedItems.find(
+          (selectedItem) =>
+            optionToValue(selectedItem) === optionToValue(option),
+        ) !== undefined
+      );
+    };
```

Comment:
It needs to be tested!
```suggestion
    const isSelected = (option: Option) => (
        !!props.selectedItems.find(
          (selectedItem) =>
            optionToValue(selectedItem) === optionToValue(option),
        )
    )
```

Reply:
It is via `expectHighlightedOptions` in the tests :p